content/authors/Rahmani/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+# Display name
+title: Rahmani
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Postdoctoral Scholar
+
+# Organizations/Affiliations
+organizations:
+- name: USC QCB
+  url: "https://dornsife.usc.edu/qcb/"
+
+# Short bio (displayed in user profile at end of posts)
+bio: I am interested in computational approaches and biophysical modeling to study genome organization.
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+
+#- icon: envelope
+#  icon_pack: fas
+#  link: 'mailto:test@example.org'
+
+- icon: twitter
+  icon_pack: fab
+  link: 
+- icon: google-scholar
+  icon_pack: ai
+  link: https://scholar.google.com/citations?user=UUYEU4UAAAAJ&hl=en&oi=ao
+- icon: github
+  icon_pack: fab
+  link: https://github.com/hadirah87
+  
+# Link to a PDF of your resume/CV from the About widget.
+# To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
+# - icon: cv
+#   icon_pack: ai
+#   link: files/cv.pdf
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+email: "rahmanin@usc.edu"
+
+# Highlight the author in author lists? (true/false)
+highlight_name: false
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Researchers
+---
+
+Hadi Rahmaninejad is a Postdoctoral Scholar in the Department of [Quantitative and Computational Biology](https://dornsife.usc.edu/qcb/) at USC.  Hadi received a Ph.D. in Physics from Virginia Tech, focusing on biological physics and soft materials. 
+
```

Comment:
feel free to extend or add links to your previous groups-- up to you

Reply:
Done. Thanks for the suggestion.